wordfreq/_chinese_mapping.py

```diff
@@ -0,0 +1,3275 @@
+SIMPLIFIED_MAP = {13423: '㑔',
```

Comment:
Hmm, okay. It takes about 125 ms to import the file, as opposed to 1.4 ms to load it from a .msgpack.gz file.